src/server/cmd/pachd/main.go

```diff
@@ -118,6 +119,9 @@ func doEnterpriseMode(config interface{}) (retErr error) {
 		log.Printf("no Jaeger collector found (JAEGER_COLLECTOR_SERVICE_HOST not set)")
 	}
 	env := serviceenv.InitWithKube(serviceenv.NewConfiguration(config))
+	if err := profileutil.StartCloudProfiler("pachyderm-pachd-enterprise", env.Config()); err != nil {
+		return errors.Wrapf(err, "error starting cloud profiler")
```

Comment:
Should startup fail if we cannot setup the profiler?

Reply:
My thought is if you specify the wrong project and it fails, you can just edit the environment variable and restart, or empty it if you can't figure it out and just want to get started without it.  The alternative is to wait a while and then never see the profiles, and wonder why.

But I'm very OK with just a log.Error here if you prefer that.

Reply:
I think I'm fine with the just set it to empty if it doesn't come up.

Reply:
I made it log an error and continue on.